src/MoBi.Core/OSPSuite.Core/CoreConstants.cs

```diff
@@ -41,5 +41,12 @@ private static string compositeNameFor(char separator, params string[] names)
 
          return nonEmptyNames.Select(x => x.Trim()).ToString($"{separator}");
       }
+
+      public static class ExpressionTypeNames
+      {
+         public static string Transporter = "Transporter";
```

Comment:
why isn't it an enum?  not sure where this is going to be used at the moment. Need to see deepr